src/docdb/tree/DocDBDocumentsTreeItem.ts

```diff
@@ -61,9 +61,20 @@ export class DocDBDocumentsTreeItem extends DocDBTreeItemBase<RetrievedDocument>
 
         if (docID || docID === "") {
             docID = docID.trim();
+            let body = { 'id': docID };
+            const partitionKey: string | undefined = this._collection.partitionKey && this._collection.partitionKey.paths[0];
+            if (partitionKey) {
+                const partitionKeyValue: string = await vscode.window.showInputBox({
+                    prompt: `The partition key is ${partitionKey}. Enter a value for the partition key`,
```

Comment:
Not sure we should assign a key value if the user enters ENTER.  What do you think?

Reply:
It currently creates the nested object with an empty string. 

Looks like empty strings aren't the same as no partition key. Thanks for the catch!